src/main/java/nl/novi/TechItEasy/exceptions/RecordNotFoundException.java

```diff
@@ -0,0 +1,10 @@
+package nl.novi.TechItEasy.exceptions;
+
+public class RecordNotFoundException extends RuntimeException {
+    public RecordNotFoundException() {
```

Comment:
In de uitwerking staat nog " private static final long serialVersionUID = 1L;" tussen regel 3 en 4. 
Geen idee waarom.